src/containers/App/index.js

```diff
@@ -52,11 +56,12 @@ export const AppContainer = ({ requestCategoriesAction }) => {
           <Switch>
             <Redirect exact from="/" to="/incident" />
             <Redirect exact from="/login" to="/manage" />
+            <Redirect exact from="/manage" to="/manage/incidents" />
```

Comment:
Does this work for `/manage/` ?

Reply:
Yeah, it does. That URL is picked up by `<Route path="/manage" component={IncidentManagementModule} />`